src/routes/safe/components/Settings/ManageOwners/AddOwnerModal/screens/OwnerForm/index.tsx

```diff
@@ -34,7 +35,7 @@ const OwnerForm = ({ classes, onClose, onSubmit }) => {
   const handleSubmit = (values) => {
     onSubmit(values)
   }
-  const owners = useSelector(safeOwnersSelector)
+  const owners = (useSelector(safeOwnersSelector) as unknown) as SafeRecordProps['owners']
```

Comment:
wdyt about correctly typing `safeOwnersSelector` instead of doing hacky things?

Reply:
well... now TS is not complaining :man_shrugging: